docs/conf.py

```diff
@@ -292,7 +292,7 @@ def ensure_typedoc_on_path():
     if shutil.which("typedoc"):
         return
     if IN_READTHEDOCS:
-        subprocess.run(["npm", "ci"], cwd="../src/js")
+        subprocess.run(["npm", "ci"], cwd="../src/js", check=False)
```

Comment:
Good catch. Let's make this `check=True` here:
```suggestion
        subprocess.run(["npm", "ci"], cwd="../src/js", check=True)
```